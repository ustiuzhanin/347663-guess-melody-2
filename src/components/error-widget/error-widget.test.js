import React from 'react';
import renderer from 'react-test-renderer';
import ErrorWidget from './error-widget.jsx';

test(`ErrorWidget's snapshot`, () => {
  const tree = renderer.create(<ErrorWidget />).toJSON();

  expect(tree).toMatchSnapshot();
});
