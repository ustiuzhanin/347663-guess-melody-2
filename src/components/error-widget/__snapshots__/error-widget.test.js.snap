// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorWidget's snapshot 1`] = `
<div
  className="wrong"
/>
`;
